waku/v2/peermanager/peer_manager.go

```diff
@@ -45,6 +47,19 @@ type PeerManager struct {
 	subRelayTopics      map[string]*NodeTopicDetails
 }
 
+// PeerSelection provides various options based on which Peer is selected from a list of peers.
+type PeerSelection int
+
+const (
+	Unknown PeerSelection = iota
+	Automatic
+	LowestRTT
```

Comment:
What do you think if we make `Automatic` the default and remove `Unknown`? that way it is not necessary to specify the peer selection criteria, and there would not be any need of having a `errors.New("unknown peer selection type specified")` error

Reply:
Good idea, will do that.